og_ui/og_ui.admin.inc

```diff
@@ -1013,8 +1033,11 @@ function og_ui_field_settings($form, &$form_state) {
     }
   }
 
+
+
   $form['bundle'] = array(
-    '#title' => t('Bundles'),
+    '#title' => t('Select one'),
```

Comment:
'Select one' is too generic for a label, in my opinion. What about 'Select a bundle' or similar, to emphasize that people may want to change the default selection?

Reply:
I'm thinking that perhaps the first item in the select box should be "Select one" but the title could perhaps say "Entity bundles". It's unusual being prompted to select something without give us a clue of what it is we are selecting. 

I know that "Bundles" is an unusual term not available anywhere in the Backdrop UI (it's "developer's lingo"). So I think originally we thought of removing that to avoid confusion. Perhaps a less obscure title may be simply "Entity" since we are being asked to select either nodes, comments or custom entities.

I have to say that the list shown in the dropdown is not very practical either. It includes all entities and bundles known to Backdrop, but some of them are hardly appropriate for adding fields. For example, OG Membership Types are listed there, and even Rules Configurations (when Rules is enabled), but it would be completely wrong to add OG fields to those! D7 has the same lack of clarity.

Reply:
@olafgrabienski - This label used to be "Bundles" which I found completely useless, because even after all the work I've done on this module, the word bundles still confuses me a bit.

@argiepiano and I went with "Select One" because we couldn't find anything better that was both accurate and understandable. 

Maybe "Select a bundle" is better than just bundles, because at least folks will understand two of the three words. I like the current wording, because it causes me to click and see what I'm selecting. Once I see the list, it makes sense. But the word bundles actually discourages me from looking, but it makes me think that I won't understand what I see anyway.

Reply:
@stpaultim - thanks for your feedback. I still think, people should be able to know the topic of a select list without opening it, so we need a specific label. And I fully agree regarding the unclear term "Bundles", so let's find a better label, maybe "Entities" (which isn't perfect but still better than Bundles).

> I'm thinking that perhaps the first item in the select box should be "Select one"

@argiepiano - Isn't something like that even the usual way? The label mentions the topic, and if no item is selected as default, you see something like "Select a value" or similar.

Reply:
@argiepiano - If you want me to change the PR, let me know what you think the final text should be for that label. OR, feel free to edit the PR yourself, I think you can do that without me.

Reply:
Thanks @stpaultim. I will make a couple of cosmetic changes and will merge this. I'd like to aim at a beta release early next week.